tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -221,6 +225,18 @@ def do_log_copy(args: CensusBuildArgs) -> bool:
     return True
 
 
+def do_sync_to_external_s3_bucket(args: CensusBuildArgs) -> bool:
```

Comment:
The naming here isn't great. Will revisit.

Reply:
do_sync_bucket_to_bucket?

Reply:
Maybe `do_sync_to_replica_s3_bucket`. I think using the term `replica` makes it more consistent.